src/renderer/wg_engine/tvgWgGeometry.h

```diff
@@ -329,15 +330,14 @@ struct WgVertexBufferInd {
     void appendMiter(const Point& v0, const Point& v1, const Point& v2, float dist1, float dist2, float halfWidth, float miterLimit) {
         Point sub1 = v1 - v0;
         Point sub2 = v2 - v1;
+        if (tvg::zero(cross(sub1, sub2))) return; // collinear
```

Comment:
please check:
```
        auto shape = tvg::Shape::gen();
        shape->moveTo(100, 100);
        shape->lineTo(400,100);
        shape->lineTo(100,100.1);
        shape->strokeFill(255, 0, 0);
        shape->strokeWidth(10);
        shape->strokeJoin(tvg::StrokeJoin::Miter);
        canvas->push(std::move(shape));
```
not exactly collinear (even when I check normalized vectors: `cross(nrm1, nrm2)`) 
but sin and cos are zero

Reply:
@mgrudzinska I think, that this solusion is nice, but it is not precise enough. Lets keep existing solution and close the PR